### PR TITLE
Describe Customization panel possibilities

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

**Comment by DelazJ** [comment]
Might need review especially for the tips to restore UI. @m-kuhn ?

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -630,18 +633,41 @@ installed that you never use and that are filling your screen.
...
-you can hide entries in the Menu bar. In |checkbox| :guilabel:`Panels`, you
-find the panel windows. Panel windows are applications that can be started and
-used as a floating, top-level window or embedded to the QGIS main window as a
-docked widget (see also :ref:`sec_panels_and_toolbars`). In the |checkbox|
-:guilabel:`Status Bar`, features like the coordinate information can be
-deactivated. In |checkbox| :guilabel:`Toolbars`, you can (de)activate the
-toolbar icons of QGIS, and in |checkbox| :guilabel:`Widgets`, you can (de)activate
-dialogs as well as their buttons.
+Ticking the |checkbox| :guilabel:`Enable customization` checkbox is the first step
+on the way of QGIS customization. This enables the toolbar and the widget
```
Comment (m-kuhn) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: on the way *to*? disclaimer: I'm not a native speaker.

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -630,18 +633,41 @@ installed that you never use and that are filling your screen.
...
+panel from which you can uncheck and thus disable some GUI items.
+
+The deactivable item can be:
```
Comment (m-kuhn) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: activatable

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 3]
```diff
@@ -630,18 +633,41 @@ installed that you never use and that are filling your screen.
...
+* a **Toolbar**: the whole bar or some of its icons
+* or any **widget** from any dialog in QGIS: label, button, combobox...
 
 With |select| :sup:`Switch to catching widgets in main application`, you
-can click on elements in QGIS that you want to be hidden and find the corresponding
-entry in Customization (see figure_customization_). You can also save your
-various setups for different use cases as well. Before your changes
-are applied, you need to restart QGIS.
+can click on an item in QGIS interface that you want to be hidden and
+QGIS automatically uncheckes the corresponding entry in Customization dialog.
```
Comment (m-kuhn) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: unchecks

Comment (m-kuhn) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: in *the* customization dialog

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 4]
```diff
@@ -630,18 +633,41 @@ installed that you never use and that are filling your screen.
...
+The modifications can also be saved in a ``.ini`` file using |fileSave|
+:sup:`Save To File` button. This is a handy way to share a common QGIS
+interface among multiple users. Just click on |fileOpen| :sup:`Load from File`
+from the destination computer in order to import the ``.ini`` file.
+You can also run :ref:`command line tools <custom_commandline>` and save various
+setups for different use cases as well.
+
+.. tip:: **Easily restore predefined QGIS**
+
+   QGIS initial GUI configuration can be retrieved using one of the methods below:
```
Comment (m-kuhn) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: The initial QGIS GUI configuration can be restored by...

**Comment by m-kuhn** [comment]
I'm not a customization specialist, but to me it looks ok.

**Comment by DelazJ** [comment]
Thanks!

**Pull request merged**